epf/FileProcessor.cpp

```diff
@@ -85,12 +85,15 @@ void FileProcessor::run()
 
     pdal::FixedPointTable t(1000);
 
-    f.prepare(t);
-    f.execute(t);
-    m_progress.update(count % CountIncrement);
+    try
+    {
+        f.prepare(t);
+        f.execute(t);
+    }
+    catch (const pdal::pdal_error&)
+    {}
```

Comment:
Same as above. If we swallow this error how is it ever relayed upstream?

Reply:
There is no mechanism for propagating exceptions.